src/components/SunriseSunset/SunriseSunset.js

```diff
@@ -6,17 +6,16 @@ import moment from 'moment';
 import 'react-calendar/dist/Calendar.css';
 
 
-const SunriseSunset = ({latitude, longitude}) => {
+const SunriseSunset = ({latitude, longitude, image, fullName, addToFutureTrips, id}) => {
 
   const [dateState, setDateState] = useState(new Date())
   const [sunRiseSet, setSunRiseSet] = useState({})
+  const [newTrip, setNewtrip] = useState({})
```

Comment:
I don't see changeDate called anywhere, is it in another component that I'm missing?

Reply:
oops - old code that got changed - thanks for catching that